extension/maxminddb.c

```diff
@@ -646,50 +632,65 @@ static PyTypeObject Reader_Type = {
     .tp_init = Reader_init,
 };
 
-static PyMethodDef Metadata_methods[] = {
-    { NULL, NULL, 0, NULL }
-};
+static PyMethodDef Metadata_methods[] = {{NULL, NULL, 0, NULL}};
 
 /* *INDENT-OFF* */
 static PyMemberDef Metadata_members[] = {
-    { "binary_format_major_version", T_OBJECT, offsetof(
-          Metadata_obj, binary_format_major_version), READONLY, NULL },
-    { "binary_format_minor_version", T_OBJECT, offsetof(
-          Metadata_obj, binary_format_minor_version), READONLY, NULL },
-    { "build_epoch", T_OBJECT, offsetof(Metadata_obj, build_epoch),
-          READONLY, NULL },
-    { "database_type", T_OBJECT, offsetof(Metadata_obj, database_type),
-          READONLY, NULL },
-    { "description", T_OBJECT, offsetof(Metadata_obj, description),
-          READONLY, NULL },
-    { "ip_version", T_OBJECT, offsetof(Metadata_obj, ip_version),
-          READONLY, NULL },
-    { "languages", T_OBJECT, offsetof(Metadata_obj, languages), READONLY,
-          NULL },
-    { "node_count", T_OBJECT, offsetof(Metadata_obj, node_count),
-          READONLY, NULL },
-    { "record_size", T_OBJECT, offsetof(Metadata_obj, record_size),
-          READONLY, NULL },
-    { NULL, 0, 0, 0, NULL }
-};
+    {"binary_format_major_version",
+     T_OBJECT,
+     offsetof(Metadata_obj, binary_format_major_version),
+     READONLY,
+     NULL},
+    {"binary_format_minor_version",
+     T_OBJECT,
+     offsetof(Metadata_obj, binary_format_minor_version),
+     READONLY,
+     NULL},
+    {"build_epoch",
+     T_OBJECT,
+     offsetof(Metadata_obj, build_epoch),
+     READONLY,
+     NULL},
+    {"database_type",
+     T_OBJECT,
+     offsetof(Metadata_obj, database_type),
+     READONLY,
+     NULL},
+    {"description",
+     T_OBJECT,
+     offsetof(Metadata_obj, description),
+     READONLY,
+     NULL},
+    {"ip_version",
+     T_OBJECT,
+     offsetof(Metadata_obj, ip_version),
+     READONLY,
+     NULL},
+    {"languages", T_OBJECT, offsetof(Metadata_obj, languages), READONLY, NULL},
+    {"node_count",
+     T_OBJECT,
+     offsetof(Metadata_obj, node_count),
+     READONLY,
+     NULL},
+    {"record_size",
+     T_OBJECT,
+     offsetof(Metadata_obj, record_size),
+     READONLY,
+     NULL},
+    {NULL, 0, 0, 0, NULL}};
 /* *INDENT-ON* */
 
 static PyTypeObject Metadata_Type = {
-    PyVarObject_HEAD_INIT(NULL, 0)
-    .tp_basicsize = sizeof(Metadata_obj),
+    PyVarObject_HEAD_INIT(NULL, 0).tp_basicsize = sizeof(Metadata_obj),
```

Comment:
Hrm, this isn't great. I saw one like this above (first file in the commit) as well, though not as weird.